melodic/distribution.yaml

```diff
@@ -6808,6 +6808,21 @@ repositories:
       url: https://github.com/ros-visualization/rqt_dep.git
       version: master
     status: maintained
+  rqt_ez_publisher:
+    doc:
+      type: git
+      url: https://github.com/OTL/rqt_ez_publisher.git
+      version: melodic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/OTL/rqt_ez_publisher-release.git
+      version: 0.4.0-1
```

Comment:
It's a bit weird that kinetic has version 0.5.0, while you are releasing 0.4.0 into melodic.  It's not disallowed, but I just wanted to make sure this is what you meant.  Is it?

Reply:
Uhhm, it might be mistake. I'll try again.